documents/2-RTB/Documentazione_Interna/Verbali_Interni/Verbale_Interno_2024-11-28.typ

```diff
@@ -0,0 +1,119 @@
+#import "/template/document.typ": *
+
+#show: project.with(
+  title: "Verbale interno del 2024-11-28",
+  recipients: (
+    p.vardanega,
+    p.cardin,
+  ),
+  changelog: (
+    "0.1",
+    "2024-11-28",
+    (p.ferazzani),
+    (p.campagnaro),
+    "Stesura del verbale",
+  ),
+)
+
+= Contenuti del verbale
+
+== Informazioni sulla riunione
+
+- *Luogo*: Discord
+- *Ora di inizio*: 15:00
+- *Ora di fine*: 16:30
+- *Partecipanti*: #p.ferazzani, #p.belenkov, #p.santi, #p.mahdi, #p.fracaro, #p.campagnaro, #p.benedetti.
+
+= Ordine Del giorno
+- Chiarimenti sugli attori del sistema
+- Scrittura decisioni conformi alle norme di progetto
+- Creazione di una tabella con tutte le #glossary("issue") programmate per lo sprint in atto.
+- Modifica procedura creazione delle #glossary("issue")
+- Modifica procedura di approvazione dei documenti
+- Modifica #glossary("Pull Request")
+// - Backlog ?
+- Creazione documento glossario e pianificazione incontri
+- Utilizzo di Jira
+- Nuovo Responsabile
+
+== Chiarimenti sugli attori del sistema
+
+L'incontro si è aperro con una discussione se l'#glossary("LLM" ) fosse o meno un attore del sistema. Dopo un'attenta analisi, basata anche sulle slide del corso e in particolare l'esempio del LogIn con servizi terzi come Facebook, si è deciso che l'#glossary("LLM") è un attore esterno in quanto risiede su un sistema separato con cui il team non può interagire se non attraverso delle richieste.
+
+== Scrittura decisioni conformi alle norme di progetto
+
+//Matte non ricordo, correggi pure quello da correggere lol.
+Il team, avendo riscontrato problemi a tracciare in maniera efficiente ed efficace le decisioni prese, ha deciso di modificarne la struttura in modo che queste rispecchino le direttive imposte dalle norme di progetto.
+
+== Creazione di una tabella con tutte le #glossary("issue") programmate per lo sprint in atto.
+
+A partire dal prossimo sprint, il responsabile si impegna a creare tutte le issue previste per lo sprint in corso e a riportarle in una tabella con le seguenti colonne:
```

Comment:
50
A partire dal prossimo sprint, il responsabile si impegna a creare tutte le issue previste per il prossimo periodo e a riportarle nella sezione Prossime attività, del verbale interno dell'incontro, in una tabella strutturata come segue:

Reply:
done